openedx/core/djangoapps/user_api/urls.py

```diff
@@ -25,9 +26,21 @@
 
 urlpatterns = patterns(
     '',
-    url(r'^v1/me$', ME, name='own_username_api'),
-    url(r'^v1/accounts/{}$'.format(settings.USERNAME_PATTERN), ACCOUNT_DETAIL, name='accounts_api'),
-    url(r'^v1/accounts$', ACCOUNT_LIST, name='accounts_detail_api'),
+    url(
```

Comment:
Please don't change code unrelated to what you're doing, even if the style is less than ideal.